library/utils/route-helpers/paths.ts

```diff
@@ -24,21 +24,21 @@ export const allClusterRoutes = [
   clusterDetailRoute,
   clusterEditRoute,
   clusterCreateRoute,
-];
+] as const;
```

Comment:
This is required for the union type to work, without this we get `type InfraRoute = string`, with this we get 
```
type InfraRoute = "clusters" | "cluster/:clusterName" | "cluster/:clusterName/edit" | "clusters/create" | "hosts" | "host/:id" | "host/:id/edit" | "register-hosts" | "/hosts/set-up-provisioning" | ... 4 more ... | "sites/:siteId"
```